### PR TITLE
2628: Changes for adding anonymous account users

### PR DESCRIPTION
**Pull request by sumesh-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/2628

*Description of changes:*
- Changes for adding anonymous account users
- Handles only positive flows - Error cases are not handled

*Checklist:*
I confirm that below checklist items are addressed in this pull request; (check the boxes applicable)
- [ ] No lint errors
- [ ] No test case failures and proper coverage for all modules/classes
- [ ] Updated the deployment configs for new environment variable(s)
- [ ] Updtaed the postman collection in entity repository (https://github.com/bcgov/entity/tree/master/api-e2e/postman) for e2e tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/models/Organization.ts [hunk 1]
```diff
@@ -19,6 +19,8 @@ export interface Organization {
   orgType?: string;
   members?: Member[];
   invitations?: Invitation[];
+  // eslint-disable-next-line camelcase
+  access_type: string;
```
Comment (sutherlanda) [inline auth-web/src/models/Organization.ts]: Maybe this is already in progress but can we convert this to camelCase in the back-end like with the other fields?

File: auth-web/src/components/auth/AddUsersForm.vue [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+        } else {
+          user.membershipType = user.selectedRole.name.toUpperCase()
+        }
+      }
+      await this.createUsers({ orgId: this.currentOrganization.id, users: this.users })
+
+      this.resetForm()
+
+      // emit event to let parent know the invite sequence is complete
+      this.$emit('add-users-complete')
```
[the remark this answers is not in the document]
Comment (sumesh-aot) [reply auth-web/src/components/auth/AddUsersForm.vue]: Its a copy-paste issue, will fix it :)

[the remark this answers is not in the document]
Comment (sumesh-aot) [reply auth-web/src/components/auth/AddUsersForm.vue]: Done

File: auth-web/src/models/Organization.ts (hunk 1 above)
Comment (sumesh-aot) [reply auth-web/src/models/Organization.ts]: Done

**Pull request merged**